source/_docs/configuration/state_object.markdown

```diff
@@ -1,33 +1,59 @@
 ---
-title: "State objects"
-description: "Describes all there is to know about state objects in Home Assistant."
+title: "State and state object"
+description: "Describes all there is to know about state and the state object in Home Assistant."
 related:
   - docs: /docs/configuration/entities_domains/
     title: Entities and domains
 ---
 
-Your {% term devices %} are represented in Home Assistant as entities. The {% term entities %} will write their current {% term state %} to the state machine for other entities/templates/frontend to access. States are a current representation of the {% term entity %}.
+Devices are represented in Home Assistant as {% term entities %}. The state of an entity (for example, if a light is on, at 50% brightness in orange) can be shown on the dashboard or be used in automations. This page looks at the concepts _state_, _state object_, and _entity state attribute_.
 
-If you overwrite a state via the states dev tool or the API, it will not impact the actual device. If the device state is being polled, it will overwrite the state in the state machine the next polling.
+## State versus state object
 
-All states will always have an entity id, a state and a timestamp when last updated and last changed.
+In Home Assistant, the state is the current representation of the {% term entity %} with all its attributes at a given moment in time. This state is recorded as a _state object_. Entities constantly keep track of their state and write it into a state object, so that other entities/templates/frontend can access it. In the example&mdash;the light is on, at 50% brightness in orange&mdash;_on_ is the actual state of the light. 50% brightness and the color are entity state attributes.
```

Comment:
> In Home Assistant, the state is the current representation of the {% term entity %} with all its attributes at a given moment in time. 

This is a bit a tricky statement since a state trigger (in automation) does NOT trigger on a attribute state.

I'd add state **object** here, to avoid ambiguity of `state.state` and state as a whole go for:

```suggestion
In Home Assistant, the state object is the current representation of the {% term entity %} with all its attributes at a given moment in time. This state is recorded as a _state object_. Entities constantly keep track of their state and write it into a state object, so that other entities/templates/frontend can access it. In the example&mdash;the light is on, at 50% brightness in orange&mdash;_on_ is the actual state of the light. 50% brightness and the color are entity state attributes.
```